setup.py

```diff
@@ -1,20 +1,16 @@
 from setuptools import setup
 from setuptools import find_packages
 
-long_description = '''
-A package for machine learning inference in FPGAs. We create firmware
-implementations of machine learning algorithms using high level synthesis
-language (HLS). We translate traditional open-source machine learning package
-models into HLS that can be configured for your use-case!
-
-For more information visit the webpage:
-https://hls-fpga-machine-learning.github.io/hls4ml/
-'''
+from os import path
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
 
 setup(name='hls4ml',
-      version='0.2.0',
+      version='0.2.2',
```

Comment:
@vloncar @Duchstf @thesps we should decide what version this should be once merged/uploaded to PyPI Maybe `0.3.0`?

Reply:
I proposed 0.3.0, because the API stuff is pretty major change to way users interact with our library.